lib/activeadmin-reform.rb

```diff
@@ -0,0 +1 @@
+require_relative 'active_admin/reform'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.